x-pack/solutions/observability/plugins/infra/public/components/asset_details/hooks/use_asset_details_url_state.ts

```diff
@@ -99,6 +100,7 @@ const AssetDetailsUrlStateRT = rt.partial({
   alertStatus: AlertStatusRT,
   dashboardId: rt.string,
   alertMetric: rt.string,
+  preferredSchema: DataSchemaFormatRT,
```

Comment:
We need to update the [locator](https://github.com/elastic/kibana/blob/main/x-pack/solutions/observability/plugins/observability_shared/common/locators/infra/asset_details_locator.ts#L45). The variable names need to match